beacon-chain/rpc/apimiddleware/custom_handlers.go

```diff
@@ -193,17 +201,53 @@ func handlePostSSZ(
 	return true
 }
 
-func sszRequested(req *http.Request) bool {
+func sszRequested(req *http.Request) (bool, error) {
 	accept, ok := req.Header["Accept"]
```

Comment:
nitpick: would be slightly more concise if you used https://pkg.go.dev/net/http#Header.Values
```
accept := req.Header.Values("Accept")
if len(accept) == 0 {
    return false, nil
}
```